### PR TITLE
adds ghetto/improvised tools

### PR DESCRIPTION
**Pull request by ghost**
download

## Changelog
:cl: Zamolxius
add: Added improvised crowbars, multitool, wirecutter, welding tool, screwdriver and wrench craftable from the crafting menu.
/:cl:

## Why It's Good For The Game
it's cool

### CODE REVIEW

**Comment by Jujumatic** [comment]
Holy crap nice. I am going to abuse the shit out of improvised multitool

File: hippiestation/code/game/objects/items/tools.dm [hunk 1]
```diff
@@ -12,7 +12,7 @@
 	else
 		return ..()
 
-// TOOLSPEED REMOVAL/equal rights
+// TOOLSPEED SETTINGS START
```
Comment (Jujumatic) [inline hippiestation/code/game/objects/items/tools.dm:15]: Changing this wont change anything rn but lol

**Comment by mikolaj007** [comment]
it costs you a headset.
so either sacrifice your own or hit up a lathe, in which case you might as well make a multitool

File: hippiestation/code/game/objects/items/tools.dm [hunk 2]
```diff
@@ -83,3 +83,52 @@
...
+	icon_state = "wrench_ghetto"
+	toolspeed = 0.3
+
+/obj/item/crowbar/ghetto
+	name = "ghetto crowbar"
+	desc = "A crude, self-wrought crowbar. Heavy."
+	icon = 'hippiestation/icons/obj/tools.dmi'
+	icon_state = "crowbar_ghetto"
+	force = 12 //same as large crowbar, but bulkier
+	w_class = WEIGHT_CLASS_BULKY
```
Comment (Jujumatic) [inline hippiestation/code/game/objects/items/tools.dm:102]: Big boy

File: hippiestation/code/game/objects/items/tools.dm [hunk 3]
```diff
@@ -83,3 +83,52 @@
...
+	toolspeed = 0.3
+	random_color = FALSE
+
+/obj/item/weldingtool/ghetto
+	name = "ghetto welding tool"
+	desc = "A MacGyver-style welder."
+	icon = 'hippiestation/icons/obj/tools.dmi'
+	icon_state = "welder_ghetto"
+	toolspeed = 0.3
+	materials = list(MAT_METAL=140)
```
Comment (Jujumatic) [inline hippiestation/code/game/objects/items/tools.dm:127]: So 14 units of fuel right

**Review by Jujumatic — DISMISSED** [review]
Very nice

**Comment by Jujumatic** [comment]
Just rob it off of someone

**Comment by k-k-karma** [comment]
No. people are going to use this in permabrig and it will be awwful

File: hippiestation/code/game/objects/items/tools.dm (hunk 3 above)
Comment (ghost) [reply hippiestation/code/game/objects/items/tools.dm:127]: no it's 20 units like a regular one, I just upped the metal reimbursement at an autolathe from 40material units

File: hippiestation/code/game/objects/items/tools.dm (hunk 1 above)
Comment (ghost) [reply hippiestation/code/game/objects/items/tools.dm:15]: yeah it just looks and sounds better

File: hippiestation/code/game/objects/items/tools.dm (hunk 3 above)
Comment (Jujumatic) [reply hippiestation/code/game/objects/items/tools.dm:127]: My bad lol

Comment (ghost) [reply hippiestation/code/game/objects/items/tools.dm:127]: it ok

**Comment by WarRage333** [comment]
Sticking metal sheets up the ass new meta, i love it

**Comment by Jujumatic** [comment]
At that rate delete asses as you can screwdriver multitool yourself out of permabrig anyway

**Comment by mikolaj007** [comment]
well, perma isnt that hard to escape from anyway, this just makes it a bit easier but also allows savvy people more freedom and mcgyver-ish moments

**Comment by Lxtoofonea** [comment]
+1 i love improvised shit

**Comment by ghost** [comment]
this is ready for merge

**Comment by Carbonhell** [comment]
what's the point of this? there are ghetto items that act like tools already, having ghetto tools as a whole kinda defeats the point of their existance doesn't it?

**Comment by mikolaj007** [comment]
Only one i can think of is using a pair of signalers instead of a multitool

**Comment by ghost** [comment]
yeah what mikolaj said, theres no ghetto crowbar, wirecutter, etc

**Comment by yoyobatty** [comment]
Maybe give them a chance to fail and break, also reduce their toolspeed a bit.

**Comment by ghost** [comment]
jujumagic review @Jujumatic

**Comment by TothBrush** [comment]
> Maybe give them a chance to fail and break, also reduce their toolspeed a bit.

chance of breaking sounds kinda cancerous but reduced toolspeed would be a really effective way to balance them.

**Comment by yoyobatty** [comment]
That sounds fair considering how easy they are to make. Perhaps not an rng chance but a use limit. Say 30-50 uses then they break, kinda like Rust.

**Comment by Jujumatic** [comment]
A use limit would be good if we want to actually go with the tools breaking. Imo however just having the tools work slow is more than enough balance, rather than making it so the tools actually break. Also I think all of the tools should be kept as small  bc I dont want big crowbar peepee

**Comment by ExpAntique** [comment]
+1, but give them some kind of downside, like really slow tool speed or a high chance to fail i.e. multitool shocking you, welder burning you/exploding, screwdriver/crowbar snapping in two, etc etc.

**Comment by DaydreamPuppeh** [comment]
+1.

**Comment by ghost** [comment]
@ExpAntique It looks like this has enough votes so Ima merge this. If they are too strong or something we can just get @zamolxius to make another PR to give em a bit of a nerf

File: hippiestation/code/game/objects/items/tools.dm (hunk 2 above)
Comment (ghost) [reply hippiestation/code/game/objects/items/tools.dm:102]: good varied design, nice job @zamolxius

File: hippiestation/code/game/objects/items/tools.dm (hunk 3 above)
Comment (ghost) [reply hippiestation/code/game/objects/items/tools.dm:127]: nice

**Review by ghost — APPROVED** [review]
Code looks good, it's creative and has well thought out drawbacks

**Pull request merged**